src/main/java/com/schedule/workout/workoutScheduler/controller/workout/CreateWorkoutModel.java

```diff
@@ -0,0 +1,91 @@
+package com.schedule.workout.workoutScheduler.controller.workout;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.schedule.workout.workoutScheduler.database.model.UserDB;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+
+public class CreateWorkoutModel {
+
+    private String id;
+    @NotEmpty
+    @Size(min=2, max=20)
+    private String name;
+    @NotEmpty
+    @Size(min=2, max=80)
+    private String description;
+    @NotNull
+    private Integer duration;
+    @NotNull
+    @Min(value = 1)
+    @Max(value = 5)
+    private Integer difficulty;
+
+
+    @ManyToOne
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JoinColumn(name = "user_id")
```

Comment:
All of these are only used on hibarnate managed models (database models). Don't need them on web models